src/frontend/src/test-e2e/util.ts

```diff
@@ -78,12 +47,6 @@ export async function runInBrowserCommon(
       "An error occurred during e2e test execution. Logs can be found in the wdio.log file and an additional error screenshot was saved under screenshots/error. On Github Actions you can find the log and screenshots under 'Artifacts'."
     );
     throw e;
-  } finally {
-    if (outer) {
-      // only close outer session
-      await browser.deleteSession();
```

Comment:
it looks like the "deleteSession()" can lead to some weird timeout issues where following tests overtake the failed one. I'd like to remove it for now. I think Selenium Server can handle a couple of old dangling sessions